BM/ras/ras_common.sh

```diff
@@ -21,3 +25,47 @@ pkg_check_install() {
         fi
     done
 }
+
+# check mcelog service is properly configured and running
+mcelog_config() {
+    local daemon=0
+    local logfile=0
+
+    pgrep -x mcelog >/dev/null 2>&1
+    if [ $? -ne 0 ]; then
+        test_print_trc "mcelog service is not running, start the service"
+        /usr/sbin/mcelog --ignorenodev --daemon --logfile=${MCELOG_LOGFILE}
+        exit 0
+    fi
+    for i in $(pgrep -a -x mcelog); do
+        if [[ $i == *"--daemon"* ]]; then
+            daemon=1
+        elif [[ $i == *"--logfile=${MCELOG_LOGFILE}"* ]]; then
+            logfile=1
+        fi
+    done
+    if [ "$daemon" -eq 0 ] || [ "$logfile" -eq 0 ]; then
+        test_print_trc "mcelog service is not properly configured, reload the service."
+        kill -9 $(pgrep -x mcelog)
+        /usr/sbin/mcelog --ignorenodev --daemon --logfile=${MCELOG_LOGFILE}
+    fi
+}
+
+# disable MCE CMCI
+disable_cmci() {
```

Comment:
Hi @laifryiee  Please use two spaces for indentation for shell code. And please refine the whole file. Thanks

Reply:
Hi, @ysun  all indentation issues are resolved. Please help take a look again.